frontend/packages/core/src/Resolver/input.tsx

```diff
@@ -126,6 +128,7 @@ const SchemaResolver = ({ schema, expanded, onClick, submitHandler }: SchemaReso
 
   return (
     <Form noValidate onSubmit={schemaValidation.handleSubmit(() => submitHandler(data))}>
+      {/* {process.env.REACT_APP_DEBUG_FORMS && <DevTool control={schemaValidation.control} />} */}
```

Comment:
Is this supposed to be commented out when the other one isnt? Not really sure what it's doing

Reply:
Yeah, I left it as a comment for now but we should figure out how to display a debug tool based on the resolver input type.